activestorage/app/controllers/active_storage/blobs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# Take a signed permanent reference for a blob and turn it into an expiring service URL for download.
```

Comment:
Active Storage is a Rails::Engine so the controllers (and their routes) are available in users' Rails apps 😄